diff --git a/hydra-gtk/src/main.c b/hydra-gtk/src/main.c
index 72d6dd7..24248a9 100644
--- a/hydra-gtk/src/main.c
+++ b/hydra-gtk/src/main.c
@@ -15,8 +15,8 @@
 #include <string.h>
 
 char *hydra_path1 = "./hydra";
-char *hydra_path2 = "/usr/local/bin/hydra";
-char *hydra_path3 = "/usr/bin/hydra";
+char *hydra_path2 = "/data/data/com.termux/files/usr/bin/hydra";
+char *hydra_path3 = "/data/data/com.termux/files/home/.local/bin/hydra";
 
 GtkWidget *wndMain;
 char *HYDRA_BIN;
